rancher2/schema_cluster_aks_config_v2.go

```diff
@@ -2,13 +2,18 @@ package rancher2
 
 import (
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
 )
 
 const (
 	clusterAKSV2Kind   = "aksV2"
 	clusterDriverAKSV2 = "AKS"
 )
 
+var (
+	clusterAKSOutboundType = []string{"loadBalancer", "managedNATGateway", "userAssignedNATGateway", "userDefinedRouting"}
```

Comment:
Optional suggestion: Not sure if these are case sensitive (probably not) but maybe we should match these to how they appear in the [Azure docs](https://learn.microsoft.com/en-us/azure/aks/egress-outboundtype#outbound-types-in-aks) i.e. `managedNatGateway` and `userAssignedNatGateway`.